core/controllers/question.py

```diff
@@ -41,3 +42,82 @@ def get(self):
         return self.render_json({
             'questions_dict': questions_dict
         })
+
+
+class QuestionsHandler(base.BaseHandler):
+    """This handler completes PUT/DELETE requests for questions."""
+    REQUIRE_PAYLOAD_CSRF_CHECK = False
```

Comment:
Why False? That seems wrong. The pattern should be similar to ExplorationPage and ExplorationHandler.

That might also address your get_json issues.

Reply:
Updated.